### DISCUSSION TITLE
Sudden Increase in RMSE_e During Initial Training

### DISCUSSION THREAD

**Discussion by XiangYu-Ji001** [deepmodeling/deepmd-kit#4777] [Q&A] [answered]
**DeePMD-kit Version：**
DeePMD-kit v3.0.3

**Problem Description:**
A sudden increase in RMSE_e is observed during the initial training process. The system consists of an aqueous NaClO₄ aqueous solution with 339 atoms. Training was conducted on 1000 frames from a single OUTCAR file, with 800 frames for training and 200 for testing. Three separate initial training runs were performed, RMSE_e showed sharp increases during or near the end of long training sequences. Is this situation normal? Could it be caused by inappropriate learning rate or loss function settings? For ease of review, I’ve attached the input.json file.
I confirm that the training set data are fully converged. The VASP parameters are set as follows: ENCUT = 400 eV, EDIFF = 1 × 10⁻⁴ eV, SIGMA = 0.2, and only the Gamma point is used for k-point sampling (simulation box side length ~15 Å).
![image](https://github.com/user-attachments/assets/7133766d-1f29-4a0a-aa87-1e696a73e5a2)
![image](https://github.com/user-attachments/assets/719a69d8-b717-4e1d-ab13-38de5b35d671)

[input.json](https://github.com/user-attachments/files/20501723/input.json)

**Comment by XiangYu-Ji001** [comment, marked as answer]
The issue has been resolved, which was caused by the low accuracy of AIMD used in the previous training set. After modifying the INCAR parameters to Ecut=600, SIGMA=0.05, EDIFF=5E-6 with 2\*2\*2 Kpoints, no sudden Increase were observed during the training process.
![fig_lcurve_2  2](https://github.com/user-attachments/assets/c7054e3d-3693-48f2-aeb6-b94591205dca)
